google/resource_bigtable_gc_policy.go

```diff
@@ -62,9 +62,15 @@ func resourceBigtableGCPolicy() *schema.Resource {
 					Schema: map[string]*schema.Schema{
 						"days": {
 							Type:        schema.TypeInt,
-							Required:    true,
+							Optional:    true,
+							Deprecated:  "Deprecated in favor of seconds",
 							Description: `Number of days before applying GC policy.`,
 						},
+						"seconds": {
+							Type:        schema.TypeInt,
+							Required:    true,
```

Comment:
We cannot add a `Required: true` in a minor version release of the Terraform provider, as it will break existing configurations.

Reply:
Indeed, I've update the code to consider this optional and throw an error if both fields aren't set